src/pages/Books.js

```diff
@@ -21,7 +21,7 @@ const Books = () => {
 
   return (
     <div className="container-div">
-      {booksList.length > 0 ? booksList.map((book) => (<Book key={book.id} book={book} />)) : ''}
+      {booksList.length > 0 ? booksList.map((book) => (<Book key={book[0]} book={book} />)) : ''}
```

Comment:
- [ ] As suggested by the previous reviewer kindly add the background color for the Books container. As you can see from the design the color of the books are different from the encompassing element. The books have a **`white`** background while their containers have a **`greyish`** background color. Kindly implement that.

  Your Implementation | Zeplin Design
  :---:|:---:
  ![mre-back-g](https://user-images.githubusercontent.com/81468675/173101695-1f53e562-48f7-4421-b3bf-2638a6654b99.jpg) | ![mre-2](https://user-images.githubusercontent.com/81468675/173101685-56541be3-907c-472f-9bd2-c1899b914498.jpg)